galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/OpenstackManagerImpl.java

```diff
@@ -209,12 +215,14 @@ public ILinuxProvisionedImage provisionLinux(String tag, OperatingSystem operati
                                 availableCapability = availableCapability.substring(1);
```

Comment:
Don't you also want to log the availableCapability when it isn't equal to the chosenCapability ? possible on line 225 ?